sigstore/_cli.py

```diff
@@ -294,15 +294,18 @@ def _verify(
 
     verified = True
     for file in files:
-        if verify(
+        result = verify(
             rekor_url=rekor_url,
             file=file,
             certificate=certificate,
             signature=signature,
             cert_email=cert_email,
-        ):
+        )
+        if result:
             click.echo(f"OK: {file.name}")
         else:
+            failure = cast(VerificationFailure, result)
+            click.echo(failure.reason)
```

Comment:
This isn't related but I just noticed that the failure reason doesn't get printed at the moment.